pages/2fa.md

```diff
@@ -1,5 +1,23 @@
 # Two-Factor Authentication
 
-!!! todo
+Two factor authentication (2FA) must be enabled on your CCR account or you will not be able to login to any CCR portals.  You will see the error `You do not have access to this resource`
 
-    write me
+## Enabling 2FA on your CCR account  
+[Follow these directions to properly enable 2FA on your CCR account](portals/idm/#enabling-two-factor-authentication)
+
+!!! Warning  
+    Your UBIT account in Duo is NOT tied to your CCR account.  You will NOT receive push notifications from your CCR account  
+
+## Logging into your CCR account with 2FA  
+
+[See these instructions for logging into CCR's portals with 2FA enabled](portals/idm/#logging-in-with-two-factor-authentication)
+
+Watch this super short video that demonstrates the login process:  
+![type:video](https://youtube.com/embed/g6hWYooFKWE)  
+
+## My 2FA Doesn't Work!  
+
+If you enabled 2FA prior to January 2023, it is possible your CCR account is not properly linked to whichever authentication app you're using.  Sometimes people forget to scan the QR code to link their account or some misunderstood and thought their CCR account was linked to their UBIT account.  If this is the case for you, you are locked out of your account.  Please contact CCR help to receive instructions on how to proceed.  For those enabling 2FA after the IDM portal update in January 2023, it will be more difficult to get in this locked account situation but if your OTP token isn't working or you lost access to the device that generates your OTP tokens, contact us!
+
+!!! Tip  
+    Changing phones? If you won't have access to both devices, disable 2FA on your CCR account prior to swapping phones. When you get your new phone [follow these instructions](portals/idm/#managing-tokens-for-devices) to link a new device and then re-enable 2FA  
```

Comment:
Fix page link, should be: `[See these instructions for logging into CCR's portals with 2FA enabled](portals/idm.md#logging-in-with-two-factor-authentication)`